bolt-design-system/bolt
# Component Status Board - Pattern Lab Component Folder Updates

## Jira
http://vjira2:8080/browse/BDS-1046

## Summary
Renames existing component folders in Pattern Lab in order to automatically match up with the names of the package names (what things are called once they are published to NPM)

## Details
Updates / renames a handful of existing Pattern Lab component folders to address naming mismatches between the @boklt/component names published to NPM vs how things are named in Pattern Lab. 

This is partially a workaround to a known bug in PL when folders don't include number prefixes + partially a workaround to keep folders visually the same thing in Pattern Lab but have URL names be automatically matched up with a component's source code.

Note: these updates heavily depend on #1083 in order to fully test and confirm changes are working as expected. The actual code changes in this PR should be specific to https://github.com/bolt-design-system/bolt/commit/8eace0093f5ed09129ad08337fa5773c2d9f0533

## How to test
- Confirm build working as expected
- Confirm component-specific URLs on the new component status board (ex. UL, OL, and Button Group) link to the correct section as expected
